app/containers/ProductsContainer.jsx

```diff
@@ -1,7 +1,7 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import Products from '../components/Products';
-import {productsActionCreator, initialState} from '../reducers/products'
+// import {productsActionCreator} from '../reducers/products'
```

Comment:
can be deleted? we don't have any need for productsActionCreator in our navbar stuff